Index: src/main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package src;\r\n\r\npublic class main {\r\n\r\n    cacacacacacacaca\r\n}\r\n
===================================================================
diff --git a/src/main.java b/src/main.java
--- a/src/main.java	(revision f2b7921b57be75e58961a0430c5276dab359c8f6)
+++ b/src/main.java	(date 1744724656546)
@@ -2,5 +2,4 @@
 
 public class main {
 
-    cacacacacacacaca
 }
